pkgs/by-name/to/tokyonight-gtk-theme/package.nix

```diff
@@ -1,12 +1,15 @@
-{
-  lib,
-  stdenvNoCC,
-  fetchFromGitHub,
-  gnome,
-  sassc,
-  gnome-themes-extra,
-  gtk-engine-murrine,
-  colorVariants ? [] # default: install all icons
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, gnome
+, sassc
+, gnome-themes-extra
+, gtk-engine-murrine
+, colorVariants ? []
+, sizeVariants ? []
+, themeVariants ? []
+, tweakVersions ? []
+, iconVariants ? []
```

Comment:
Please use `nixfmt-rfc-style` for formatting.

Reply:
> Please use `nixfmt-rfc-style` for formatting.

How must it be used? I never used it.

Reply:
`nixfmt-rfc-style` provides binary `nixfmt` you can just invoke it with filepath

Reply:
Oh nice: https://github.com/NixOS/nixfmt so just:
```
nixfmt <path-to-package.nix>
```